packages/client/src/Views/ChannelTabsContainer.jsx

```diff
@@ -169,6 +172,17 @@ class ChannelTabsContainer extends Component {
     const [ state ] = this.context;
     let force = state.allForces.find((force) => force.uniqid === state.selectedForce);
 
+    const channelsArray = Object.values(state.channels);
+    const isOnlyMap = channelsArray.filter(channel => channel.name.toLowerCase() === "mapping").length === 1;
+
+    if (channelsArray.length === 1 && isOnlyMap) {
+      return <Mapping currentTurn={state.currentTurn} role={state.selectedRole} currentWargame={state.currentWargame} selectedForce={state.selectedForce} allForces={state.allForces} allPlatforms={state.allPlatformTypes} phase={state.phase} channelID={"map"} imageTop={imageTop} imageBottom={imageBottom} imageLeft={imageLeft} imageRight={imageRight}></Mapping>
+    }
```

Comment:
@jcdiprose - I'm pretty sure there's a nested if here:
```
if(channel.len == 1) {
   if(isOnlyMap) {
     return <Mapping>
   } else {
     return <Channel>
  }
}
```

Reply:
Either or, I prefer the seperate if statements, would you prefer it nested?